boards/BMS/Src/App/states/App_InitState.c

```diff
@@ -19,6 +20,10 @@ static void InitStateRunOnEntry(struct StateMachine *const state_machine)
     App_SharedClock_SetPreviousTimeInMilliseconds(clock, App_SharedClock_GetCurrentTimeInMilliseconds(clock));
     App_Accumulator_InitRunOnEntry(accumulator);
     App_OkStatus_Enable(bms_ok_status);
+
+    // AIR+ opens upon entering init state
+    // Should never be open at this point, this is only for redundancy
+    App_Airs_OpenAirPositive(airs);
```

Comment:
Do you mean "Should always be open at this point"? Otherwise this isn't redundant